crate_universe/src/rendering.rs

```diff
@@ -674,7 +677,8 @@ impl Renderer {
                         &dependency.id.version,
                         &dependency.target,
                     );
-                    aliases.insert((label, alias.clone()), configuration.clone());
+                    // Should `Select` support `BTreeMap<Label, T>` or `BTreeMap<U, T>`?
```

Comment:
The above change would be easy enough, but then how to update `SelectDict<T>` is also a question.

Reply:
I think you probably have better instincts on this than I do... WDYT?

It doesn't seem awful to keep all the keys here as `String`s; introducing either a generic generic, or a "bounded to just `String` or `Label`" generic all seem pretty reasonable.